### ISSUE TITLE
animate before scroll

### ISSUE THREAD

**Issue by erbill** [developedbyed/3d-scroll-ball#1]
Thanks for the explanatory video! I'm trying to make animation start when user enters the website like say from 1.jpg to 25.jpg and stays there, only after scrolling it acts as usual, and stops at 25.jpg when scrolled at the top of the page, not where the animation starts. What kind of app.js modifications would you suggest?

**Comment by Saikiasan** [comment]
> Thanks for the explanatory video! I'm trying to make animation start when user enters the website like say from 1.jpg to 25.jpg and stays there, only after scrolling it acts as usual, and stops at 25.jpg when scrolled at the top of the page, not where the animation starts. What kind of app.js modifications would you suggest?

Yeah this idea is great but you should know that the script will load after the page elements load in.

Be sure to use async function.

And then make sure to place a click or scroll trigger for the preload animation
